app_dart/lib/src/request_handlers/dart_internal_subscription.dart

```diff
@@ -0,0 +1,32 @@
+// Copyright 2023 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:cocoon_service/src/request_handling/body.dart';
+import 'package:meta/meta.dart';
+
+import '../../cocoon_service.dart';
+import '../request_handling/subscription_handler.dart';
+import '../service/logging.dart';
+
+/// An endpoint for listening to build updates for dart-internal builds and
+/// saving the results to the datastore.
+///
+/// The PubSub subscription is available here:
+/// https://pantheon.corp.google.com/cloudpubsub/subscription/detail/dart-internal-build-results-sub?project=flutter-dashboard
```

Comment:
Nit: link to the external url: https://console.cloud.google.com/cloudpubsub/subscription/detail/dart-internal-build-results-sub?project=flutter-dashboard

Seems the links in other `subscription` APIs broke (a `console.` prefix is needed). Could you help fix them or created a doc todo bug?

Reply:
Updated, as well as updated the other links missing the `console.` prefix, and a couple of README's also using the `pantheon` prefix